hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -301,6 +301,19 @@ public void printReport() {
     messages.forEach(print);
   }
 
+  /**
+   * Print out the messages for HadoopNestedDirGenerator
```

Comment:
As this is base class, please remove the inherited class details here.

you can write something like,
"Print out reports with the given message."

Reply:
Changed the javadoc.